packages/table/src/Table.ts

```diff
@@ -447,6 +447,7 @@ export class Table extends SizedMixin(SpectrumElement, {
             this.tableBody = this.querySelector('sp-table-body') as TableBody;
             if (!this.tableBody) {
                 this.tableBody = document.createElement('sp-table-body');
+                this.tableBody.tabIndex = 0;
```

Comment:
Is this needed in the case that the items do not force the Table Body to scroll?